graphs/search/bfs.go

```diff
@@ -2,14 +2,14 @@
 // reference: https://en.wikipedia.org/wiki/Tree_traversal
 package search
 
-/*BreadthFirstSearch is an algorithm for traversing or searching graph data structures.
+/*BFS Breadth-first search is an algorithm for traversing or searching graph data structures.
```

Comment:
Perhaps `and` instead of `or`?
Would we ever traverse the graph without searching?
Would we ever search the graph without traversing?

Reply:
this sentence is quoted from Wikipedia. Of course, traversing and searching are equals.